.github/workflows/newbuild-vcpkg.yml

```diff
@@ -0,0 +1,333 @@
+          sudo apt-get -yq install -f mono-complete
+
+      - name: OSX Dependencies
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform && contains(matrix.os, 'macos') }}
+        shell: "bash"
+        run: |
+          brew install bison flex pkg-config autoconf autoconf-archive automake libtool cmake openjdk@11
+
+      - name: "Setup NuGet credentials"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        shell: "bash"
+        run: |
+          ${{ matrix.mono }} `vcpkg fetch nuget | tail -n 1` \
+            sources add \
+            -source "https://nuget.pkg.github.com/hpcc-systems/index.json" \
+            -storepasswordincleartext \
+            -name "GitHub" \
+            -username "${{ github.actor }}" \
+            -password "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: "Remove builtin vcpkg"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: .
+        shell: "bash"
+        run: |
+          ${{ matrix.sudo }} rm -rf "$VCPKG_INSTALLATION_ROOT"
+
+      - name: Check out source code
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: "vcpkg Bootstrap"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: .
+        shell: "bash"
+        run: |
+          ./vcpkg/bootstrap-vcpkg.sh
+
+      - name: "Generate HPCC-Platform Build Files"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: .
+        shell: "bash"
+        run: |
+          mkdir build
+          cd build
+          cmake .. -DSKIP_ECLWATCH=ON -DCONTAINERIZED=OFF -DUSE_OPTIONAL=OFF ${{ matrix.cmake_config_options }}
+
+      - name: "Remove vcpkg build files"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: ./vcpkg
+        shell: "bash"
+        run: |
+          rm -rf ./buildtrees
+          rm -rf ./downloads
+
+      - name: "Build HPCC-Platform"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: ./build
+        shell: "bash"
+        run: |
+          cmake --build . ${{ matrix.cmake_build_options }}
+
+      - name: "Build HPCC-Platform"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: ./build
+        shell: "bash"
+        run: |
+          cmake --build . ${{ matrix.cmake_build_options }}
+
+      - name: Bundle Package  
+        if: github.event_name == 'push' && github.ref == 'refs/tags/*'
```

Comment:
as defined, this action will only trigger on "push" afaics, so condition is superflous ?

Reply:
Its only on a pushed tag...

Reply:
IOW when a tag is pushed we build the package and upload it as an asset to the tag...  (I can't fully test until this is in master...)

Reply:
yes this comment was connected with the other, in that I can only see how the action even would ever be "push" at the moment (because only on: push:)